addons/website_sale_delivery/views/website_sale_delivery.xml

```diff
@@ -50,5 +50,52 @@
         </xpath>
     </template>
 
+    <template id="country_state_shipping" name="Shipping Country and states" inherit_id="website_sale.checkout">
+        <xpath expr="//select[@name='shipping_country_id']" position="replace">
+            <select name="shipping_country_id" class="form-control" t-att-disabled="'disabled' if shipping_id &gt;= 0 else ''">
+                <option value="">Country...</option>
+                <t t-foreach="shipping_countries or []" t-as="country">
+                    <option t-att-value="country.id" t-att-selected="country.id == checkout.get('shipping_country_id')"><t t-esc="country.name"/></option>
+                </t>
+            </select>
+        </xpath>
+        <xpath expr="//select[@name='country_id']" position="replace">
+            <select name="country_id" class="form-control js_country_all" t-att-style="not shipping_id and 'display:none' or ''" t-att-disabled="not shipping_id and True or False">
```

Comment:
I would find a better class name for js_country_all (and similar) like o_shipping_countries ?